### DISCUSSION TITLE
At my first submission, they rejected my application bc the proof is used

### DISCUSSION THREAD

**Discussion by sheepal** [community/community#64527] [GitHub Education]
### Hi everyone,

Hello.
I uploaded my ID card at first but got rejected because of a lack of the date.
So I pay some money to print my enrollment proof document and submitted it.
but after a moment it just said "Your uploaded proof has already been used previously. ".
I didn't use it before, an error occurred AT FIRST submission of the document.
I don't know what to do next, there's no support for these cases. need help, thank you.

**Comment by arafajs** [comment]
The error message "Your uploaded proof has already been used previously" means that GitHub has already received the document you are trying to upload. This could be because you have already submitted the document for verification, or it could be because someone else with the same name and email address has already submitted the document.

To resolve this issue, you will need to contact GitHub Support. They will be able to investigate the issue and help you to get your verification approved.

Here are the steps on how to contact GitHub Support:

- Go to the GitHub Support website: https://github.com/contact.
- Click on the "Contact Us" button.
- Select the "GitHub Education" category.
- Fill out the form and provide as much information as possible about your issue.
- Click on the "Submit" button.

GitHub Support will usually respond to your request within 24 hours.

**Reply by sheepal** [reply]
Thank you for answering. I tried to contact the support but they don't provide help about these cases. (involved with GitHub Education) Therefore I couldn't fill up or submit something, and even contact for proper support.
I struggled with the education submission, and at the first moment when I uploaded the doc I'd already had 5th rejection. does it be the reason for the error message? I didn't submit it before, but I got the message right after the submission😥
